README.md

```diff
@@ -354,6 +354,37 @@ Checkout the new branch to continue working:
 (master)$ git checkout my-branch
 ```
 
+<a name="keep-whole-file"></a>
+### I want to keep the whole file from another ref-ish
+
+Say you have a working spike, with hundreds of changes. Everything is working. Now, you commit into another branch to save that work.
```

Comment:
Also, maybe add a short - one clause? - description of spike. Doesn't have to be the whole paragraph above, just might help some people. Thanks for helping me learn something new!